libs/utils/utils.js

```diff
@@ -1006,10 +1021,13 @@ async function processSection(section, config, isDoc) {
 
 export async function loadArea(area = document) {
   const isDoc = area === document;
+  let resolveDeferredFn;
 
   if (isDoc) {
+    window.milo = {};
```

Comment:
Nit: should we declare this inside `setupDeferredPromise`, since we don't really use it here? Also, should it be future-proofed a bit by declaring it as `window.milo = window.milo || {}`? Maybe this global namespace property will be used by other pieces of logic than happen before, this would just override everything.

Reply:
I encountered an issue with doing `window.milo ??= {}` because I was testing on the milo homepage which has:

`<h1 id="milo">milo</h1>`

This means that `window.milo` actually is populated and points to that element.  So now when running the above code, `window.milo` isn't an empty object, but an h1 element.

By declaring it here, I'm pretty sure that there wouldn't be anyplace that declares `window.milo` before this runs.

Also we want to be very strict about what we allow in this object.  It should *only* be used for values that have to be shared cross-script.  In this case it's for martech to be able to get the deferred status.